lib/iris/__init__.py

```diff
@@ -156,21 +156,27 @@ def __init__(self, cell_datetime_objects=True, netcdf_promote=True,
         If Iris code is executed with multiple threads, note the values of
         these options are thread-specific.
 
-        The option `cell_datetime_objects` controls whether the
-        :meth:`iris.coords.Coord.cell()` method returns time coordinate
-        values as simple numbers or as time objects with attributes for
-        year, month, day, etc. In particular, this allows one to express
-        certain time constraints using a simpler, more transparent
-        syntax, such as::
+        .. deprecated:: 2.0.0
+
+            The option `cell_datetime_objects` is deprecated and will be
+            removed in a future release. `cell_datetime_objects` is set
+            to True by default and should not be altered.
+
+            The option `cell_datetime_objects` controlled whether the
+            :meth:`iris.coords.Coord.cell()` method would return time
+            coordinate values as simple numbers or as time objects with
+            attributes for year, month, day, etc. In particular, this allows
```

Comment:
"allows" --> "allowed"? Maybe? This "allows" behaviour is still true, of course...

How about:
"Cells are now represented as time objects by default, allowing you to express time constraints using a simpler syntax. For example: ..."